Guided Investigation - WAF data.ipynb

```diff
@@ -0,0 +1,4156 @@
+{
```

Comment:
<p>If you don't need to show the full contents of the cols you could experiment with this pandas option</p><p><code>display.max_colwidth</code> sets the maximum width of columns. Cells of this length or longer will be truncated with an ellipsis.</p><pre class="ql-syntax" spellcheck="false">In [51]: df = pd.DataFrame(np.array([['foo', 'bar', 'bim', 'uncomfortably long string'],
   ....:                             ['horse', 'cow', 'banana', 'apple']]))
   ....: 

In [52]: pd.set_option('max_colwidth', 40)

In [53]: df
Out[53]: 
       0    1       2                          3
0    foo  bar     bim  uncomfortably long string
1  horse  cow  banana                      apple

In [54]: pd.set_option('max_colwidth', 6)

In [55]: df
Out[55]: 
       0    1      2      3
0    foo  bar    bim  un...
1  horse  cow  ba...  apple

In [56]: pd.reset_option('max_colwidth')
</pre><p> </p><p>Note you can also temporarily apply an option:</p><pre class="ql-syntax" spellcheck="false">In [20]: with pd.option_context("display.max_rows", 10, "display.max_columns", 5):
   ....:     print(pd.get_option("display.max_rows"))
   ....:     print(pd.get_option("display.max_columns"))
   ....: 
10
5
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>